src/fs/root/tree.ts

```diff
@@ -92,13 +100,13 @@ export default class RootTree implements Puttable {
     // Load private bits
     const privateCID = links[Branch.Private]?.cid || null
 
-    let mmpt, privateTree
+    let mmpt, privateTrees
     if (privateCID === null) {
       mmpt = await MMPT.create()
-      privateTree = await PrivateTree.create(mmpt, key, null)
+      privateTrees = await makeNewPrivateTrees(keys, mmpt)
```

Comment:
is there a scenario in which the app is creating the private tree & also needs to permission & create multiple sub-trees?

Reply:
oh is this for scaffolding the initial private tree structure? (ie in Drive?)

Reply:
wait but it also errors if the parnetNameFilter is not available :thinking: So wouldn't we only be able to create the one?

Reply:
This needs more testing, but essentially it:
1. Sorts the keys alphabetically. So if you have a key for the root, it'll always create the root tree first.
2. Find the parent name filters for creating sub trees. It does this by either looking in indexeddb, or by finding a parent tree and then getting the namefilter that way.

> oh is this for scaffolding the initial private tree structure? (ie in Drive?)

Yes. Also replacing the old `empty`

Reply:
ahhh ok cool cool yeah that makes sense :ok_hand: